gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/transformation/Transformation.java

```diff
@@ -101,7 +104,7 @@ public LG execute(LG graph) {
    */
   @SuppressWarnings("unchecked")
```

Comment:
Remove this suppression here and add `<>` to all `new TransformX<(...)` calls below